infinispan-remote/persistence/sql-store/src/main/resources/sqlQueryCache.xml

```diff
@@ -0,0 +1,28 @@
+<distributed-cache>
+    <encoding media-type="application/x-protostream"/>
+    <persistence>
+        <query-jdbc-store dialect="H2"
+                          shared="false"
+                          read-only="true"
+                          key-columns="id">
+            <connection-pool connection-url="jdbc:h2:tcp://localhost:9123/~/library;"
+                             username="infinispan"
+                             password="secret"
+                             driver="org.h2.Driver"/>
+            <schema message-name="Book" package="library"/>
```

Comment:
Was embedded key not required here?

Reply:
No, since I use it as the key of the cache without doing key/value. but maybe I will add a key mapping example in books too.